### DISCUSSION TITLE
Unable to download the mesh agent from the RMM

### DISCUSSION THREAD

**Discussion by ti-padraodofonseca** [amidaware/tacticalrmm#1883] [General]
![image](https://github.com/amidaware/tacticalrmm/assets/69602461/41842a9d-5da5-426e-aca1-fab4c2bff9eb)

Mesh site: https://mesh.******.com.br/
Mesh username: sgwnhvwn
Mesh token: d3decaffad18eb***************
Mesh device group: TacticalRMM
Auth token ok: N6nv9tMdX21s****************
Mesh url ok: ws://127.0.0.1:4430/control.ashx?auth=***************
Mesh device id ok: JEAl@*************************

**Comment by wh1te909** [comment]
what's the output of `cat /etc/hosts` from your trmm server?

**Comment by ti-padraodofonseca** [comment]
pdfadmin@pdf-rmm-001:~$ cat /etc/hosts
127.0.0.1 localhost
127.0.1.1 pdf-rmm-001 apitactical.*********.com.br rmm.*********.com.br mesh.*********.com.br

/# The following lines are desirable for IPv6 capable hosts
::1     ip6-localhost ip6-loopback
fe00::0 ip6-localnet
ff00::0 ip6-mcastprefix
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters

**Comment by ti-padraodofonseca** [comment]
Após realizar o update do Tactical e a atualização do certificado para Certbot, antes utilizado Wildcartd próprio.

O mesmo funcionou corretamente desde então. Obrigado!
